anti-analysis/packer/huan/packed-with-huan.yml

```diff
@@ -4,7 +4,9 @@ rule:
     namespace: anti-analysis/packer/huan
     authors:
       - jakub.jozwiak@mandiant.com
-    scope: file
+    scopes:
+      static: file
+      dynamic: file
```

Comment:
good shortcut here may be to only update the scopes and don't even worry about the features for `file` scope rules